com.hack_computer.Clubhouse.json

```diff
@@ -0,0 +1,276 @@
+{
+    "app-id": "com.hack_computer.Clubhouse",
+    "add-extensions": {
+        "com.hack_computer.Clippy.Extension": {
+            "version": "master",
+            "directory": "clippy",
+            "no-autodownload": false,
+            "autodelete": true
+        }
+    },
+    "branch": "master",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "eos-clubhouse",
+
+    "finish-args": [
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--env=GNOTIFICATION_BACKEND=gtk",
+        "--env=GSETTINGS_SCHEMA_DIR=/run/host/usr/share/glib-2.0/schemas/",
+        "--env=GTK3_MODULES=/app/clippy/lib/libclippy-module.so",
+        "--filesystem=/var/lib/AccountsService/icons/:ro",
+        "--filesystem=/var/lib/flatpak/exports/share/applications:ro",
+        "--filesystem=/var/lib/flatpak/exports/share/icons:ro",
+        "--filesystem=host",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=~/.local/share/flatpak",
+        "--nofilesystem=~/.local/lib/python3.5",
+        "--own-name=com.endlessm.GameStateService",
+        "--own-name=com.hack_computer.Clubhouse",
+        "--own-name=com.hack_computer.GameStateService",
+        "--own-name=com.hack_computer.HackSoundServer",
+        "--own-name=com.hack_computer.HackToolbox",
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--socket=session-bus",
+        "--socket=wayland",
+        "--socket=x11",
```

Comment:
we prefer fallback-x11 here if wayland is available these days as that prefers wayland where available.